.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -1,10 +1,14 @@
-﻿if ($Env:RUNNER_OS -eq "Windows")
+if ($Env:RUNNER_OS -eq "Windows")
 {
     choco install sql-server-express --no-progress
 }
 else
 {
     docker pull mcr.microsoft.com/mssql/server &&
-    docker run --name sql2019 -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=Password1!' -p 1433:1433 -d 'mcr.microsoft.com/mssql/server:2019-latest' &&
-    docker exec -u 0 sql2019 bash -c 'mkdir /data; chmod 777 /data -R; chown mssql:root /data'
+    docker run --name sql2019 `
+        --env 'ACCEPT_EULA=Y' `
+        --env 'SA_PASSWORD=Password1!' `
+        --publish 1433:1433 `
+        --detach 'mcr.microsoft.com/mssql/server:2019-latest' &&
+    docker exec --user 0 sql2019 bash -c 'mkdir /data; chmod 777 /data -R; chown mssql:root /data'
```

Comment:
Use the long name for `-c`, too.

Reply:
I really wanted to, but there doesn't seem to be a long name for that. I just noticed that I missed `chmod -R` though.

Reply:
👀 

I was referring to the lack of a long-name option for `-c` - I would expect `--command` as an equivalent, which is kind of today's standard (at least in the .NET world, I guess).

Reply:
Oh right, sorry. Bash is a couple years newer, but it's dedicated to have a POSIX mode and be fully backwards compatible with Bourne shell (`sh`) which is even older than GNU itself (43 years old). Who needs such a legacy mode today? No idea.

Reply:
🧾 Thanks for the free history lesson! 😎